ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/activity/TutorialActivity.kt

```diff
@@ -29,14 +29,16 @@ import org.cru.godtools.tutorial.databinding.TutorialActivityBinding
 private const val ARG_PAGE_SET = "pageSet"
 private const val ARG_FRMT_ARGS = "formatArgs"
 
+const val START_TUTORIAL = 102
+
 fun Context.buildTutorialActivityIntent(pageSet: PageSet, formatArgs: Bundle? = null) =
     Intent(this, TutorialActivity::class.java)
         .putExtra(ARG_PAGE_SET, pageSet)
         .putExtra(ARG_FRMT_ARGS, formatArgs)
 
 @JvmOverloads
 fun Activity.startTutorialActivity(pageSet: PageSet, stringArgs: Bundle? = null) =
-    startActivity(buildTutorialActivityIntent(pageSet, stringArgs))
+    startActivityForResult(buildTutorialActivityIntent(pageSet, stringArgs), START_TUTORIAL)
```

Comment:
this change shouldn't be made here, this should be handled within `ToolDetailsFragment`.

it will be something like:
```kotlin
startActivityForResult(context.buildTutorialActivityIntent(), REQUEST_TUTORIAL_TIPS)
```